docs/cli/quick-start-guide.md

```diff
@@ -0,0 +1,21 @@
+## Quick Start — API Platform Controller (ap)
+
+This short guide explains how to use a prebuilt `ap` binary so you can run the CLI from any location.
+
+- Install the binary: put the file on your `PATH`. For example:
+
+	- macOS / Linux:
+
+		```bash
+		sudo mv ap /usr/local/bin/
+		```
+
+	- Windows:
+
+		Place `ap.exe` in a folder that's included in your `%PATH%` (for example `%USERPROFILE%\bin`) or add its folder to your PATH.
```

Comment:
We can ask to to extract the zip to a location, then set the path to that location, without doing a sudo mv